Signal/src/view controllers/CallViewController.swift

```diff
@@ -519,7 +519,6 @@ class CallViewController: UIViewController, CallDelegate {
 
     func updateCallUI(callState: CallState) {
         let textForState = localizedTextForCallState(callState)
-        Logger.info("\(TAG) new call status: \(callState) aka \"\(textForState)\"")
```

Comment:
I took a different approach in #1614 with https://github.com/WhisperSystems/Signal-iOS/pull/1614/commits/0797df19b97b6bf2fd7999012aea5f74cdaf7d8c

Thoughts?